js/components/portal-dashboard/question-area.tsx

```diff
@@ -48,8 +49,10 @@ export class QuestionArea extends React.PureComponent<IProps>{
         </div>
         <div className={`${css.questionTextArea} ${!useMinHeight ? css.minHeight : ""}`} data-cy="question-content">
           <div className={css.questionText}>
-            {prompt ? striptags(prompt.replace(/&nbsp;/g,' ')) : ""}
+          {prompt ? renderHTML(prompt) : ""}
+            {/* {prompt ? striptags(prompt.replace(/&nbsp;/g,' ')) : ""} */}
```

Comment:
Is this commented piece obsolete?  Probably want to remove it.  Also want to tab in the `{prompt ? renderHTML(prompt) : ""}` line so it is properly aligned.

Reply:
And you can rewrite simply as `{ prompt && renderHTML(prompt) }`

Reply:
Looking  more closely at the story spec, the drawing prompt goes first.  This should be `{ drawingPrompt && renderHTML(drawingPrompt ) }`